account_avatax/models/avalara_salestax.py

```diff
@@ -146,8 +146,15 @@ def _get_avatax_supported_countries(self):
     )
     use_so_partner_id = fields.Boolean(
         string="Use Sale Customer Code on Invoice",
-        help="If Boolean is checked, SO Partner Customer Code "
-        "on Invoice will be used",
+        help="Use Sales Order's Customer field to determine Taxable "
+        "Status on the Customer Invoice. If no Sales Order exists, "
+        "Customer field on the invoice form view will be used instead",
+    )
+    hide_excemption = fields.Boolean(
+        "Hide Exemption & Tax Based on shipping address",
+        default=True,
```

Comment:
Let's keep the preexisting behavior
```suggestion
        default=False,
```